src/sentry/static/sentry/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -476,11 +227,151 @@ const AddWidgetWrapper = styled('a')`
   justify-content: center;
 `;
 
-const WidgetGhost = styled('div')`
-  display: block;
-  position: absolute;
-  cursor: grabbing;
-  opacity: 0.8;
-  box-shadow: ${p => p.theme.dropShadowHeavy};
-  border-radius: ${p => p.theme.borderRadius};
-`;
+type SortableWidgetProps = {
+  widget: Widget;
+  dragId: string;
+  isEditing: boolean;
+  isDragging: boolean;
+  onDelete: () => void;
+  onEdit: () => void;
+};
+
+function SortableWidget(props: SortableWidgetProps) {
```

Comment:
should we move this component to a separate file?

Reply:
I moved it.